pkg/operator2/deployment.go

```diff
@@ -121,18 +122,32 @@ func defaultDeployment(
 							LivenessProbe:            livenessProbe(),
 							TerminationMessagePath:   "/dev/termination-log",
 							TerminationMessagePolicy: corev1.TerminationMessagePolicy("File"),
-							//Resources: corev1.ResourceRequirements{
-							//	Requests: map[corev1.ResourceName]resource.Quantity{
-							//		corev1.ResourceCPU:    resource.MustParse("2G"),
-							//		corev1.ResourceMemory: resource.MustParse("2G"),
-							//	},
-							//},
+							Resources: corev1.ResourceRequirements{
+								Requests: map[corev1.ResourceName]resource.Quantity{
+									corev1.ResourceCPU:    resource.MustParse("10m"),
+									corev1.ResourceMemory: resource.MustParse("50Mi"),
+								},
+							},
 						},
 					},
 					// deploy on master nodes
 					NodeSelector: map[string]string{
 						"node-role.kubernetes.io/master": "",
 					},
+					Affinity: &corev1.Affinity{
+						// spread out across master nodes rather than congregate on one
+						PodAntiAffinity: &corev1.PodAntiAffinity{
+							PreferredDuringSchedulingIgnoredDuringExecution: []corev1.WeightedPodAffinityTerm{{
+								Weight: 100,
+								PodAffinityTerm: corev1.PodAffinityTerm{
+									LabelSelector: &metav1.LabelSelector{
+										MatchLabels: defaultLabels(),
+									},
+									TopologyKey: "kubernetes.io/hostname",
+								},
+							}},
+						},
+					},
 					Tolerations: []corev1.Toleration{{
```

Comment:
I think this should be same as https://github.com/openshift/console-operator/blob/master/pkg/console/subresource/deployment/deployment.go#L121-L128

Reply:
I'm trying to get some clarity on this. From the cluster I launched this morning, here's what I get:
```
Operation  Effect      Key                                   Namespace                                    Resource
Exists     NoSchedule  node.kubernetes.io/not-ready          openshift-sdn                                daemonset/sdn-controller
Exists     NoSchedule  node.kubernetes.io/not-ready          openshift-network-operator                   deployment/network-operator
Exists     NoSchedule  node.kubernetes.io/memory-pressure    openshift-monitoring                         deployment/cluster-monitoring-operator
Exists     NoSchedule  node.alpha.kubernetes.io/unreachable  openshift-machine-api                        deployment/clusterapi-manager-controllers
Exists     NoSchedule  node.alpha.kubernetes.io/notReady     openshift-machine-api                        deployment/clusterapi-manager-controllers
Exists     NoSchedule  node-role.kubernetes.io/master        openshift-sdn                                daemonset/sdn-controller
Exists     NoSchedule  node-role.kubernetes.io/master        openshift-network-operator                   deployment/network-operator
Exists     NoSchedule  node-role.kubernetes.io/master        openshift-machine-config-operator            deployment/machine-config-operator
Exists     NoSchedule  node-role.kubernetes.io/master        openshift-machine-config-operator            deployment/machine-config-controller
Exists     NoSchedule  node-role.kubernetes.io/master        openshift-machine-config-operator            daemonset/machine-config-server
Exists     NoSchedule  node-role.kubernetes.io/master        openshift-machine-config-operator            daemonset/machine-config-daemon
Exists     NoSchedule  node-role.kubernetes.io/master        openshift-machine-api                        deployment/machine-api-operator
Exists     NoSchedule  node-role.kubernetes.io/master        openshift-machine-api                        deployment/clusterapi-manager-controllers
Exists     NoSchedule  node-role.kubernetes.io/master        openshift-machine-api                        deployment/cluster-autoscaler-operator
Exists     NoSchedule  node-role.kubernetes.io/master        openshift-image-registry                     deployment/image-registry
Exists     NoSchedule  node-role.kubernetes.io/master        openshift-image-registry                     deployment/cluster-image-registry-operator
Exists     NoSchedule  node-role.kubernetes.io/master        openshift-image-registry                     daemonset/node-ca
Exists     NoSchedule  node-role.kubernetes.io/master        openshift-console-operator                   deployment/console-operator
Exists     NoSchedule  node-role.kubernetes.io/master        openshift-console                            deployment/console
Exists     NoSchedule  node-role.kubernetes.io/master        openshift-cluster-samples-operator           deployment/cluster-samples-operator
Exists     NoSchedule  node-role.kubernetes.io/etcd          openshift-machine-config-operator            daemonset/machine-config-daemon
Exists     NoSchedule  CriticalAddonsOnly                    openshift-machine-api                        deployment/clusterapi-manager-controllers
Exists     NoSchedule  -                                     openshift-monitoring                         daemonset/node-exporter
Exists     NoExecute   node.alpha.kubernetes.io/unreachable  openshift-machine-api                        deployment/clusterapi-manager-controllers
Exists     NoExecute   node.alpha.kubernetes.io/notReady     openshift-machine-api                        deployment/clusterapi-manager-controllers
Exists     NoExecute   node-role.kubernetes.io/master        openshift-machine-api                        deployment/clusterapi-manager-controllers
Exists     NoExecute   CriticalAddonsOnly                    openshift-machine-api                        deployment/clusterapi-manager-controllers
Exists     NoExecute   -                                     openshift-monitoring                         daemonset/node-exporter
Exists     -           -                                     openshift-svcat-controller-manager-operator  deployment/openshift-svcat-controller-manager-operator
Exists     -           -                                     openshift-svcat-apiserver-operator           deployment/openshift-svcat-apiserver-operator
Exists     -           -                                     openshift-service-cert-signer                deployment/service-serving-cert-signer
Exists     -           -                                     openshift-service-cert-signer                deployment/configmap-cabundle-injector
Exists     -           -                                     openshift-service-cert-signer                deployment/apiservice-cabundle-injector
Exists     -           -                                     openshift-service-ca-operator                deployment/openshift-service-ca-operator
Exists     -           -                                     openshift-sdn                                daemonset/sdn
Exists     -           -                                     openshift-sdn                                daemonset/ovs
Exists     -           -                                     openshift-operator-lifecycle-manager         deployment/packageserver
Exists     -           -                                     openshift-operator-lifecycle-manager         deployment/olm-operator
Exists     -           -                                     openshift-operator-lifecycle-manager         deployment/catalog-operator
Exists     -           -                                     openshift-multus                             daemonset/multus
Exists     -           -                                     openshift-marketplace                        deployment/marketplace-operator
Exists     -           -                                     openshift-kube-scheduler-operator            deployment/openshift-kube-scheduler-operator
Exists     -           -                                     openshift-kube-controller-manager-operator   deployment/kube-controller-manager-operator
Exists     -           -                                     openshift-kube-apiserver-operator            deployment/kube-apiserver-operator
Exists     -           -                                     openshift-ingress-operator                   deployment/ingress-operator
Exists     -           -                                     openshift-dns-operator                       deployment/dns-operator
Exists     -           -                                     openshift-dns                                daemonset/dns-default
Exists     -           -                                     openshift-core-operators                     deployment/openshift-service-cert-signer-operator
Exists     -           -                                     openshift-controller-manager-operator        deployment/openshift-controller-manager-operator
Exists     -           -                                     openshift-controller-manager                 daemonset/controller-manager
Exists     -           -                                     openshift-cluster-version                    deployment/cluster-version-operator
Exists     -           -                                     openshift-cluster-storage-operator           deployment/cluster-storage-operator
Exists     -           -                                     openshift-cluster-node-tuning-operator       deployment/cluster-node-tuning-operator
Exists     -           -                                     openshift-cluster-node-tuning-operator       daemonset/tuned
Exists     -           -                                     openshift-cluster-machine-approver           deployment/machine-approver
Exists     -           -                                     openshift-cloud-credential-operator          deployment/cloud-credential-operator
Exists     -           -                                     openshift-authentication-operator            deployment/openshift-authentication-operator
Exists     -           -                                     openshift-authentication                     deployment/openshift-authentication
Exists     -           -                                     openshift-apiserver-operator                 deployment/openshift-apiserver-operator
Exists     -           -                                     openshift-apiserver                          daemonset/apiserver
```

Reply:
@derekwaynecarr @smarterclayton what do we need here?

Reply:
@derekwaynecarr said operators should be using the tolerate all taints one.

Reply:
Which one is that exactly?  Does console need to update?

Reply:
should be: 
```yaml
tolerations:
-  operator: Exists
```
so yeah, console and the other ones need to update theirs.

Reply:
cc @spadgett @benjaminapetersen ^